DirectEmployers/MyJobs
# Add "view analytics" activity to reporting entry in topbar [PD-3293]

## What's this PR do?
Allows users with "view analytics" but not "read partner" to access reporting.
## How should this be manually tested?
Create a role to play around with. Add "read partner". Note that analytics doesn't show up in the report list. Add "view analytics". Note that both entries appear. Remove "read partner". Note that PRM reporting is gone and only analytics remains.
